articles/synapse_docker_registry.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Synapse Docker Registry
+layout: article
+excerpt: The basics of using the synapse docker registry
+---
+
+##Synapse Docker Registry
+Synapse now has its own docker registry which will allow you to have private docker repositories.  Please click [here](https://www.docker.com/products/overview) to learn how to use docker for mac, windows or linux. Below is a brief tutorial on how to use the synapse docker registry using the command line.
```

Comment:
https://www.docker.com/products/docker-registry

Docker should be capitalized everywhere that it's not explicitly referring to the function.